### ISSUE TITLE
ListView item loses role attribute on cancel

### ISSUE THREAD

**Issue by ag-petrov** [telerik/kendo-ui-core#5381]
### Bug report

When an item inside the listview is being edited and the user clicks cancel the role attribute of the listview item is lost. More information about this problem can be found in ticket - 1436557

### Current behavior
The role attribute of the item is lost

### Expected/desired behavior
The role attribute of the item should be persisted
